data-es6.js

```diff
@@ -3981,7 +3983,9 @@ exports.tests = [
         } catch(e) {}
         return true;
       */},
-      res: temp.destructuringResults,
+      res: Object.assign({}, temp.destructuringResults, {
+        closure:     false,
```

Comment:
current status of "destructuring - throws on null and undefined" is wrong. it's not supported.